media/css/firefox/whatsnew/whatsnew-nightly.scss

```diff
@@ -23,6 +23,10 @@ $image-path: '/media/protocol/img';
     color: $color-white;
 }
 
+.c-page-header-logo-fx {
+    background-image: none;
```

Comment:
This was because the original background was visible behind the Nightly one, we evidently never noticed because they just lined up. But the doge's ears are positioned differently and an orange ear was poking out.